src/Models/Services/Utils.ts

```diff
@@ -688,3 +691,51 @@ export function areResourceValuesEqual(
         return value1 === value2;
     }
 }
+
+export function getRoleIdsFromRoleAssignments(
```

Comment:
it'd be ideal if we could add some UTs for anythign going into the utils file since it's already standalone and easy to test. Maybe something for another rev.

Reply:
okay adding it now for the methods I added, then we can merge